README.md

```diff
@@ -23,35 +23,8 @@ More languages/tools will come in the future.
 
 ## Usage
 
-- Install yeoman and the generator: `yarn global add yo @thetribe/generator-project`
-- Create a new directory for your project and go to it
-
-### Guided mode
-
-Run `yo @thetribe/project` and follow the prompts to generate your project
-
-### Manual mode
-
-Run `yo @thetribe/project:root` to generate a bare root project and then
-[add more components to your project](#add-more-components)
-
-### Add more components
-
-You can add more components to an existing project with the following commands:
-- `yo @thetribe/project:create-react-app [name]`
-- `yo @thetribe/project:express [name]`
-- `yo @thetribe/project:next-js [name]`
-
-### Push your project
-
-- Create a new GitHub repository with the name you specified in the configuration
-- Create an initial commit in your project folder
-- And push it with `git push --set-upstream origin master`
-
-## If you encounter an issue while using the generator
-
-If you encounter any issue while using the generator or contributing on it please create a issue in this repository.
+You can use the generator by following the [quickstart guide](doc/QUICKSTART.md)
 
 ## Contributing
 
-You can contribute to this repository by following the [contributing guide](CONTRIBUTING.md)
+You can contribute to this repository by following the [contributing guide](doc/CONTRIBUTING.md)
```

Comment:
The contributing file should either be at the root of the repository, in the `.github` folder of the `docs` folder to be picked up by GitHub so this changes breaks it.

Reply:
Interesting didn't know that - therefore I'll just put a link into the `Quickstart` to link to the root `CONTRIBUTING`